pkg/sql/conn_executor.go

```diff
@@ -1699,6 +1699,7 @@ func (ns *prepStmtNamespace) resetTo(
 // transaction event, resetExtraTxnState invokes corresponding callbacks
 // (e.g. onTxnFinish() and onTxnRestart()).
 func (ex *connExecutor) resetExtraTxnState(ctx context.Context, ev txnEvent) {
+	ex.extraTxnState.numDDL = 0
```

Comment:
was this always incrementing forever?

Reply:
Yes, this was always incrementing for ever before :(